core/urls.py

```diff
@@ -1,25 +1,75 @@
+import inspect
+
 from django.conf.urls import include
 from django.contrib import admin
+from django.db.models import fields
+from django.http import QueryDict
 from django.urls import path
+from django_filters.rest_framework import DjangoFilterBackend
 from djproxy.urls import generate_routes
+from rest_framework import parsers, viewsets
+from rest_framework.parsers import MultiPartParser
 from rest_framework.routers import SimpleRouter
 
+from . import serializers
 from . import views
 
 
+class MultiPartJsonParser(MultiPartParser):
+    def parse(self, stream, media_type=None, parser_context=None):
+        result = super().parse(stream, media_type=media_type, parser_context=parser_context)
+
+        data = {}
+        arrays = {}
+
+        model = None
+        if parser_context and 'view' in parser_context:
+            model = parser_context['view'].get_serializer_class().Meta.model
+        for key, value in result.data.items():
+            """
+                Handle ManytoMany field data, parses lists of comma-separated integers that might be quoted.
+                eg. "1,2"
+            """
+            if hasattr(model, key) and isinstance(
```

Comment:
This `if` can be somewhat simplified: `if isinstance(getattr(model, key, None), fields.related_descriptors.ManyToManyDescriptor):`, since `None` won't be an instance of `ManyToManyDescriptor`.

Reply:
Done!